src/saleor_app/endpoints.py

```diff
@@ -21,7 +21,8 @@
 
 
 async def manifest(request: Request, settings=Depends(get_settings)):
-    manifest = settings.manifest.dict()
+    manifest = settings.manifest.dict(by_alias=True)
+    manifest["appUrl"] = ""
```

Comment:
I'm not sure but you wanted to leave this here?

Reply:
For now yes. We don't give an appUrl endpoint yet (appUrl works the same as configurationUrl). 
I believe, in future PRs we will also include support for appUrl